examples/electricity/amount_energy_from_voltage_time_resistance.py

```diff
@@ -27,45 +30,33 @@
 initial_temperature_kelvin = to_kelvin_quantity(initial_temperature)
 final_temperature_kelvin = to_kelvin_quantity(final_temperature)
 
-ohm_law_applied = ohm_law.law.subs({
-    ohm_law.current: operate_power.current,
-    ohm_law.resistance: joule_lenz_law.resistance,
-    ohm_law.voltage: joule_lenz_law.voltage
+water_mass = solve(
```

Comment:
Next time avoid doing major refactoring in examples and plots. They are merely for illustration purpose. We want many laws to work together in one example, not to have the most effective way of solving an example.

Reply:
Well I wanted to get rid of the law that featured mass and specific heat capacity so on this account I needed to replace other usages of the former law.